app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -295,6 +306,7 @@ class BrowserTabViewModel(
 
     fun onViewReady() {
         url?.let {
+            isDomainSameAsUseOurAppDomain(it)
```

Comment:
The idea is to send a pixel when a user visits the site for the first time without taking into account navigating within the same site in the same app session.

When the app launches we want to count that as a visit if the site was already opened because is effectively a new app session. That's why we need this call in the `onViewReady` method. 

Eventually the `pageChanged` method will be called but both the new url and the old url would be the same and we need them to be different to count it as a new visit (user was somewhere else and just navigated to the site). If the old url at this point was null we could then remove the method from `onViewReady` but it is not null because uses the site stored in the tab.

Reply:
I've changed the name of the method so it better explains what it does

Reply:
ok got it. We want to send `m_uoa_vad` every time the user goes to FB but not when user is browsing inside FB.

So expected behaviors is to send pixels: When we go to FB from home page or from another website. 
I guess back and forward navigation can trigger the pixel if user goes through other websites in his tab navigation history.

Reply:
If you mean `NavigateBack` and `NavigateForward` then not really. If you are in "example.org" and go to "facebook.com" that doesn't trigger a `NavigateForward` command.

The expected behaviour is to send a pixel whenever the user goes to FB but only the first time. If that's from a shortcut or creating a new tab or any other it doesn't matter, the pixel should be sent once every time FB is opened for the first time.

For example: 
1. Go to FB -> sends pixel
1. Go to example.org -> does not send pixel
1. Go to FB again -> sends pixel

Also FYI, is not just `m_uoa_vad` that gets sent, it depends on where the user is in the flow. We also have `m_uoa_vas` and `m_uoa_van`. But we only send one out of those 3 or none, in order it would be:
1. `m_uoa_van` -> User saw initial notification
1. `m_uoa_vas` -> User added a shortcut
1. `m_uoa_vad` -> User saw delete cta